docs/integrations/vercel-ai-sdk.mdx

```diff
@@ -81,6 +81,8 @@ npm install @mem0/vercel-ai-provider
 
      > `getMemories` will return raw memories in the form of an array of objects, while `retrieveMemories` will return a response in string format with a system prompt ingested with the retrieved memories.
 
+     > `getMemories` is an object with two keys: `results` and `relations` if `graph_memory` is enabled. Otherwise, it will return an array of objects.
```

Comment:
```suggestion
     > `getMemories` is an object with two keys: `results` and `relations` if `enable_graph` is enabled. Otherwise, it will return an array of objects.
```